tests/test_http.py

```diff
@@ -265,6 +265,20 @@ def test_parse_options_header(self):
                                          '        '
                                          'text/x-dvi; q=0.8, text/x-c') == \
             ('text/plain', {'q': '0.5'})
+        # Issue #932
+        assert http.parse_options_header(
+            'form-data; '
```

Comment:
It doesn't have to be, because option headers can be anything, e.g. a mimetype or a filename.